xtask/src/vm.rs

```diff
@@ -0,0 +1,115 @@
+//
+// Copyright 2020 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+use std::path::Path;
+
+use strum::IntoEnumIterator;
+use strum_macros::{Display, EnumIter};
+
+use crate::internal::*;
+
+#[derive(Debug, Display, Clone, PartialEq, EnumIter)]
+enum Variant {
+    Uefi,
+    Baremetal,
+}
+
+impl Variant {
+    pub fn enabled(&self) -> bool {
+        !matches!(self, Variant::Baremetal)
```

Comment:
Does this not work?

```suggestion
        self != Variant::Baremetal
```

Reply:
That was a bit too obvious a solution...